test/test_ensemble_builder/test_ensemble.py

```diff
@@ -27,821 +28,706 @@
-            ensemble_nbest=2,
+            ensemble_nbest=ensemble_nbest,
+            performance_range_threshold=performance_range_threshold,
             max_models_on_disc=None,
-            )
-        self._compare_read_preds(ensbuilder2.read_preds, ensbuilder.read_preds)
-
-
-class EnsembleSelectionTest(unittest.TestCase):
-    def testPredict(self):
-        # Test that ensemble prediction applies weights correctly to given
-        # predictions. There are two possible cases:
-        # 1) predictions.shape[0] == len(self.weights_). In this case,
-        # predictions include those made by zero-weighted models. Therefore,
-        # we simply apply each weights to the corresponding model preds.
-        # 2) predictions.shape[0] < len(self.weights_). In this case,
-        # predictions exclude those made by zero-weighted models. Therefore,
-        # we first exclude all occurrences of zero in self.weights_, and then
-        # apply the weights.
-        # If none of the above is the case, predict() raises Error.
-        ensemble = EnsembleSelection(ensemble_size=3,
-                                     task_type=BINARY_CLASSIFICATION,
-                                     random_state=np.random.RandomState(0),
-                                     metric=accuracy,
-                                     )
-        # Test for case 1. Create (3, 2, 2) predictions.
-        per_model_pred = np.array([
-            [[0.9, 0.1],
-             [0.4, 0.6]],
-            [[0.8, 0.2],
-             [0.3, 0.7]],
-            [[1.0, 0.0],
-             [0.1, 0.9]]
-        ])
-        # Weights of 3 hypothetical models
-        ensemble.weights_ = [0.7, 0.2, 0.1]
-        pred = ensemble.predict(per_model_pred)
-        truth = np.array([[0.89, 0.11],  # This should be the true prediction.
-                          [0.35, 0.65]])
-        self.assertTrue(np.allclose(pred, truth))
-
-        # Test for case 2.
-        per_model_pred = np.array([
-            [[0.9, 0.1],
-             [0.4, 0.6]],
-            [[0.8, 0.2],
-             [0.3, 0.7]],
-            [[1.0, 0.0],
-             [0.1, 0.9]]
-        ])
-        # The third model now has weight of zero.
-        ensemble.weights_ = [0.7, 0.2, 0.0, 0.1]
-        pred = ensemble.predict(per_model_pred)
-        truth = np.array([[0.89, 0.11],
-                          [0.35, 0.65]])
-        self.assertTrue(np.allclose(pred, truth))
-
-        # Test for error case.
-        per_model_pred = np.array([
-            [[0.9, 0.1],
-             [0.4, 0.6]],
-            [[0.8, 0.2],
-             [0.3, 0.7]],
-            [[1.0, 0.0],
-             [0.1, 0.9]]
-        ])
-        # Now the weights have 2 zero weights and 2 non-zero weights,
-        # which is incompatible.
-        ensemble.weights_ = [0.6, 0.0, 0.0, 0.4]
-
-        with self.assertRaises(ValueError):
-            ensemble.predict(per_model_pred)
-
-
-class SingleBestTest(unittest.TestCase):
```

Comment:
Are these tests still in the new version?

Reply:
Hi Matthias, yes. In detail:
```
old_test_name->new_test_name
test_get_identifiers_from_run_history_log_loss->test_get_identifiers_from_run_history
test_get_identifiers_from_run_history_accuracy->test_get_identifiers_from_run_history
```

I am using the new fixtures, so that is why the name changed